src/main/java/core/model/match/MatchKurzInfo.java

```diff
@@ -269,8 +269,16 @@ public final void setMatchDate(java.lang.String m_sMatchDate) {
 					this.matchDateTimestamp = new Timestamp(simpleFormat.parse(m_sMatchDate).getTime() + duration);
 				  }
 				catch (Exception e) {
-						HOLogger.instance().log(getClass(), e);
-					 }
+					try {
+						// Hattrick
+						SimpleDateFormat simpleFormat = new SimpleDateFormat(
+								"yyyy-MM-dd");
+						this.matchDateTimestamp = new Timestamp(simpleFormat
+								.parse(m_sMatchDate).getTime());
+					} catch (Exception ex) {
+						HOLogger.instance().log(getClass(), ex);
+					}
+				}
```

Comment:
ideally we should have separate commit / PR for separate issue ...

Reply:
isn't #514 the separate issue? we discussed it on slack. see this https://github.com/akasolace/HO/commit/eaa1b9b246a9d791049df3adfd0b41afd1178459#commitcomment-39510796

the commited change is one possibility to repare it. Another one yould be to update the database and introduce long time formats.

Reply:
yes this is related to #514, when I closed that issue I introduced the problem because I did not know/thought that very old entries could be missing time information.
What you just did here fix the issue I introduced but it does not directly relate to this PR.  But no problem leave it here, it is a small one and it does not pollute the PR a lot